hack/run-functests.sh

```diff
@@ -22,14 +22,24 @@ export CLEAN_PERFORMANCE_PROFILE="false"
 # Latency tests env variables
 export LATENCY_TEST_RUN=${LATENCY_TEST_RUN:-false}
 
-if [ "$FEATURES" == "" ]; then
-	echo "[ERROR]: No FEATURES provided"
-	exit 1
+echo "Running local tests"
+
+
+if [ "$DONT_FOCUS" == true ]; then
```

Comment:
should we put a default in this file for the `$DONT_FOCUS` variable?

Reply:
if the variable is not defined, the condition above will be false anyway.
If you think it would be more readable, I'll add it